src/Products/Annotation/Config/AnnotationConfiguration.cs

```diff
@@ -11,25 +12,64 @@ namespace GroupDocs.Annotation.MVC.Products.Annotation.Config
     /// </summary>
     public class AnnotationConfiguration : CommonConfiguration
     {
-        private string FilesDirectory = "DocumentSamples/Annotation";      
-        private string DefaultDocument = "";
-        private int PreloadPageCount = 0;
+        [JsonProperty]
+        private string filesDirectory = "DocumentSamples/Annotation";
+
+        [JsonProperty]
+        private string defaultDocument = "";
+
+        [JsonProperty]
+        private int preloadPageCount = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 'preloadPageCount', the compiler will do that for you.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)